modules/standard/Math.chpl

```diff
@@ -634,7 +637,10 @@ module Math {
      ``mod(x,y) = x - y * floor(x / y)``.
 
      If the arguments are of unsigned type, then
-     fewer condititionals will be evaluated at run time. 
+     fewer condititionals will be evaluated at run time.
```

Comment:
(old) spell check on "conditionals".  Also I'm not seeing what changed on this line, did you remove an extra space or something?